src/test/java/BaseTest.java

```diff
@@ -10,4 +21,30 @@ public static void chromeConfigs() {
         }
     }
 
-}
+    @BeforeClass
+    @Parameters({"BaseURL"})
+    public void launchBrowser(String BaseURL) {
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.addArguments("--disable-notifications");
+
+        driver = new ChromeDriver(chromeOptions);
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
```

Comment:
**Needless Repetition!** You can remove these variable because they are in the `class BasePage` 

`waitForVisibilityOfElement`   and  `waitForElementToBeClickable`